### PR TITLE
Do not remove the file when file attribute value is deleted

### PR DESCRIPTION
**Pull request by IKarbowiak**
Do not delete the file from storage when an attribute value is deleted.
This is the temporary solution, the way of keeping attribute files will be changed. We need to track what is uploaded.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6171 AttributeValueDelete attempts to delete files from untrusted user inputs leading to directory traversal](https://app.clickup.com/t/2549495/SALEOR-6171)

**Review by maarcingebala — APPROVED** [review]

**Review by jakubkuc — APPROVED** [review]

**Review by korycins — APPROVED** [review]

File: saleor/core/tasks.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 
 @app.task
 def delete_from_storage_task(path):
-    default_storage.delete(path)
+    if default_storage.exists(path):
+        default_storage.delete(path)
```
Comment (NyanKiyoshi) [inline saleor/core/tasks.py:15]: Django is already silently skipping the file if it doesn't exist upon deletion failure

**Review by NyanKiyoshi — CHANGES_REQUESTED** [review]

File: saleor/core/tasks.py (hunk 1 above)
Comment (IKarbowiak) [reply saleor/core/tasks.py:15]: I drop removing file URL when the attribute value is deleted.

Comment (NyanKiyoshi) [reply saleor/core/tasks.py:15]: The `exists()` check can be removed as Django is already skipping inexistent files, it will save a 3rd party call

**Review by maarcingebala — APPROVED** [review]

**Review by NyanKiyoshi — APPROVED** [review]

**Pull request merged**